character.py

```diff
@@ -0,0 +1,147 @@
+
+from wordle import playWordle
+from items import taser1
+from items import fists1
+from items import item1
+class Character:
+
+#Represents a character in the game.
+
+#Args:
+    #name (str): The name of the character.
+    #health (int): The initial health of the character.
+
+    def __init__(self, name, health):
+        self.name = name
+        self.health = health
+
+NPC = Character("Diddy", 100)
+Player = Character("", 100)
+Dragon = Character("Dragon", 100)
+
+
+
+def talk_to_player():
```

Comment:
These are great helper methods - but aren't quite class methods. They would need to be indented into the actual character class. And - these methods really aren't functions of the character, so it might not make sense to put them in the character class to begin with. I think it might make sense to put these in either a utils file to keep the main file clean (basically a file with general helper methods) or just straight into the main runner file.

Reply:
Adding a player method might look like def update_health(item) or change_name(new_name) or introduce_myself() etc. Basically behaviors that the character has or can do.